wire-ios/Wire-iOS/Sources/UserInterface/Conversation Options/Conversation+OptionsConfiguration.swift

```diff
@@ -130,7 +130,7 @@ extension ZMConversation {
             }
         }
 
-        func fetchConversationLink(completion: @escaping (Result<String?, Error>) -> Void) {
+        func fetchConversationLink(completion: @escaping (Result<(uri: String?, secured: Bool), Error>) -> Void) {
```

Comment:
Should the naming be `uri` or `url`?

Reply:
Following the naming in Swagger that's why I sticked with uri.